### ISSUE TITLE
Invalid option -M

### ISSUE THREAD

**Issue by khismatullin** [lukaszcz/pascaladt#2]
steps:

```
ADT_VERSION=1.0.1
cd ~/Downloads
wget https://codeload.github.com/lukaszcz/pascaladt/tar.gz/refs/tags/v$ADT_VERSION -O pascaladt-$ADT_VERSION.tar.gz
tar xvf pascaladt-$ADT_VERSION.tar.gz
sudo rm -rf pascaladt-$ADT_VERSION.tar.gz
cd pascaladt-$ADT_VERSION
make
sudo make install
```

error (when installing):

```
make dynamic RELEASE=true OPTS= VER=-1.0.1 MCP_OPTS=-dMCP_REAL -dMCP_CARDINAL -dMCP_POINTER
make: invalid option -- 'M'
```

**Comment by lukaszcz** [comment]
I'll fix this in the future. For now, just use the default options in the installation script.

**Comment by lukaszcz** [comment]
I believe this is solved in the 1.0.2 release.
